src/category_theory/whiskering.lean

```diff
@@ -15,7 +15,7 @@ variables (C : Type u₁) [𝒞 : category.{v₁} C]
           (E : Type u₃) [ℰ : category.{v₃} E]
 include 𝒞 𝒟 ℰ
 
-def whiskering_left : (C ⥤ D) ⥤ ((D ⥤ E) ⥤ (C ⥤ E)) :=
+@[simps] def whiskering_left : (C ⥤ D) ⥤ ((D ⥤ E) ⥤ (C ⥤ E)) :=
```

Comment:
Again I'm unsure about the wisdom of having overlapping simp lemmas (`whiskering_left_obj_map` below).

Reply:
Reverted.